ctx.go

```diff
@@ -817,6 +817,21 @@ func (c *DefaultCtx) Locals(key any, value ...any) any {
 	return value[0]
 }
 
+// Locals function utilizing Go's generics feature.
+// This function allows for manipulating and retrieving local values within a request context with a more specific data type.
+func Locals[V any](c Ctx, key any, value ...any) V {
```

Comment:
should value be of type V? that way you can call local without the type param and it increases type safety a bit.

`fiber.Locals(c, "age", 50)`

Reply:
ok i'll change it

Reply:
yes good point

Reply:
@ryanbekhen can you look at this

```suggestion
func Locals[V any](c Ctx, key any, value ...V) V {
```

Reply:
@ReneWerner87 do you want to change it to `func Locals[V any](c Ctx, key any, value ...V) V`